cmd/release/gh-release/main.go

```diff
@@ -26,18 +26,30 @@ var (
 // TODO: implement number override for release
 func main() {
 	branch := flag.String("branch", "", "Release branch, e.g. 1-22")
+	overrideNumber := flag.String("overrideNumber", "", "Optional override number, e.g. 1")
```

Comment:
Should this be named something like "override version" to distinguish that it is a string and not a number?

Reply:
Release number is an established thing throughout the code and release process. I would be hesitant to change this to version because I can imagine that there would be confusion between this and Kubernetes version.